src/testRunner/unittests/publicApi.ts

```diff
@@ -182,34 +182,3 @@ describe("unittests:: Public APIs:: getChild* methods on EndOfFileToken with JSD
     assert.equal(endOfFileToken.getChildCount(), 1);
     assert.notEqual(endOfFileToken.getChildAt(0), /*expected*/ undefined);
 });
-
-describe("unittests:: Public APIs:: sys", () => {

```

Comment:
Worth keeping?

Reply:
We’ll bring it back when we decide what the correct thing to assert is